examples/unity/README.md

```diff
@@ -0,0 +1,85 @@
+# Insertable Streams Unity sample
+This sample application shows how to integrate insertable streams (video) with unity.
+Integrating this sample with **Vonage Video SDK** will allow you to publish a unity scene as your video. 
+
+
+# Demo
+You can see a demo of this sample running at
+
+## Running the App
+
+### Prerequisite:
+ - Unity - this sample was built with Unity *2021.3.6f1* (make sure WebGL is installed)
+
+### Building Unity:
+
+ - From Unity Hub select *Open -> Add Project From Disk* and select **UnityVideoProject**
+ - Select *File->Build Settings* and select Platform **WebGL** and click **Switch Platform**
+ - Click **Build** 
+ - On the popup window *Save As* **demoBuild** (this name is a must since we use this name in the demo to load Unity)
+ - After the build is finished copy **all** files from *demoBuild->Build* to *Examples->Unity->public* (public folder is not part of the repo you should create it) the files are:
+	- demoBuild.wasm
+	- demoBuild.loader.js
+	- demoBuild.framework.js
+	- demoBuild.data
+	
+### Building the app:
+ - Open your terminal at: *Examples->Unity->public* run this commands
+	 - *npm install*
+	 - *npm run dev* (for local debug run) will run on *http://localhost:3000/*
+	 - *npm run build* -> *npm run preview* (for local release run) will run on *http://localhost:4173/*
```

Comment:
For local testing the last command is not relevant.

Reply:
i didnt understand what did you mean so i didnt change anything